internal/provider/provider.go

```diff
@@ -91,10 +82,6 @@ func New(ctx context.Context, config *Opts, podWatcher kubernetes.PodResourceMan
 	case "debian", "ubuntu":
 		p.pkgManager = new(ospkg.DebianManager)
 
-		// Check if we're root and otherwise skip this step.
-		if os.Geteuid() != 0 {
```

Comment:
Well, will this really not be needed if `MapRoot != 0`?

Reply:
why, systemk runs as root?

Reply:
Oh right.